### DISCUSSION TITLE
resolutions greater than 360p, audioformat = uknown

### DISCUSSION THREAD

**Discussion by luizenrike** [omansak/libvideo#259] [General]
Hello, I'm trying to download videos with resolutions higher than 360p. The download occurs, however, the video is muted, this happens because the audioformat for videos with resolutions greater than 360p are in unknown format.

![1](https://user-images.githubusercontent.com/84132822/198293638-b2ec6867-f60f-4f1e-bb33-dd0f063bdc06.PNG)
![3](https://user-images.githubusercontent.com/84132822/198293662-6df86875-b128-46d5-8285-30a84fdbb690.PNG)
![2](https://user-images.githubusercontent.com/84132822/198293681-03022b0b-c996-44d5-8943-1163f56323c4.PNG)

**Comment by hondek** [comment]
These FormatCode's including only video stream, you need to merge with audio track.

Here is FormatCode's table:
https://gist.github.com/sidneys/7095afe4da4ae58694d128b1034e01e2

**Reply by luizenrike** [reply]
In fact, I think you misunderstood. The problem is not in the formatcode but in the audio format, which returns unknown, and audioformat is read-only, i can not modified property in which case your would have to check that, right?

**Reply by hondek** [reply]
You can check which formatcodes include audio track, most of formatcodes have only video or audio stream.
